---
layout: page
title: Lab Members
permalink: /members
---

### RJ Nowling

<img src="/images/photos/nowling.jpg" width="25%" height="25%" />

Email: nowling@msoe.edu<br />[CV](/static/rnowling_resume.pdf)

I've been an Associate Professor in the Electrical Engineering & Computer Science department of the [Milwaukee School of Engineering](https://www.msoe.edu) since August 2018.  From 2014 to 2018, I worked on scalable machine learning systems at [AdRoll](http://www.adroll.com/) and [Red Hat](http://www.redhat.com/). Before then, I received a Ph.D. in [Computer Science & Engineering](http://cse.nd.edu/) from the [University of Notre Dame](http://www.nd.edu) and a B.S. in Computer Science and Mathematics from [Eckerd College](http://www.eckerd.edu).  My research over the last few years has employed statistics and machine learning to study the genomic data from insect vectors.

## **Current Students**

### John P. (Winter 2019 - Present)
John has been a member of the lab since Winter 2019.  John is working on integrating multi-omics data to identify and annotate putative functional components of enhancer sequences.

### Sam K. (Summer 2022 - Present)
Sam is evaluating techniques for clustering enhancers based on the correlation of their variants to identify enhancers impacted by the same evolutionary processes.

### Alex M. (Summer 2022 - Present)
Alex is evaluating graph learning and other approaches for developing permutation-invariant ML models.  Alex's work is a prerequisite for creating supervised models for inversion detection and segmentation.

### Sumedha S. (Fall 2022 - Present)
Sumedha is working on creating a dashboard for the multi-omics analsysis tool that will allow identifying sequences of interest for further study.

## **Alumni**

### Oliver B. (Winter 2018-2019)
Oliver was an exchange student from the [Czech Technical University in Prague](https://www.cvut.cz/en) studying at MSOE for the 2018-2019 school year.  Oliver was working on applying convolutional neural networks to detecting prostates in Magnetic Resonance (MR) images.

### Rafael R. G. (Fall - Spring 2019-2020)
Rafael graduated from the Computering Engineering program at MSOE in Spring 2020.  During the 2019-2020 academic year, Rafael applied machine learning to identifying and predicting enhancer properties of sequences.

### Ben H. (Winter and Spring 2019-2020)
Ben graduated from the Software Engineering program at MSOE in Spring 2020.  During the Winter and Spring of 2019-2020, Ben implemented and evaluated published CNN models on enhancer sequence prediction.

### Krystal M. (Spring 2019 - Spring 2020)
Krystal graduated from the [BioMolecular Engineering](https://www.msoe.edu/academics/undergraduate-degrees/engineering/biomolecular-engineering/) program at MSOE in 2019. Krystal worked in the lab from the Spring 2019 quarter through the end of the the summer of 2019. Krystal evaluated methods for detecting chromsomal inversions from SNPs and then implemented and ran a variant calling pipeline.  Krystal started a [M.S. in Bioinformatics](https://www.marquette.edu/grad/programs-bioinformatics.php) at [Marquette University](https://www.marquette.edu/) in Fall 2019.

### Errin M. (Fall and Winter 2020-2021)
Errin worked on characterizing enhancer sequences in the *Anopheles* genome.

### Andrew R. (Fall and Winter 2020-2021)
Andrew analyzed differences between FAIRE peaks for versions 3 and 5 of the *Aedes* genome.

### Matt A. (Summer 2019, Fall 2020 - Spring 2021)
Matt worked with me during the summer of 2019 and is back for the 2020-2021 academic year.  Matt is a junior in the [Computer Science](https://www.msoe.edu/academics/undergraduate-degrees/engineering/computer-science/) program at MSOE.  Matt previously implemented a population genetics simulator and explored how different processes of molecular evolution present in PCA of SNP data.  This year, Matt is working on automating parameter selection for the inversion detection using machine learning.

### Jarred K. (Summer 2021 - Spring 2022)
Jarred evaluated ensemble techniques for handling imbalanced genomics data in classification problems.

### José H. (Summer 2022)
José prototyped a GUI application for annotating peaks in functional genomics data.  José's tool will make it easier to curate high-quality training sets for the next generation of ML-based peak callers.

### Rachel M. (Summer 2022)
Rachel, a biology major from nearby Alverno College, analyzed distributions of enhancers and genes in the _Anopheles coluzzii_ mosquito genome.

### Daniel L. (Fall 2020 - Summer 2022)
Daniel worked on a variety of projects related inversion detection with k-mer count features.

